snapm/command.py

```
@@ -1081,7 +1353,56 @@ def print_snapsets(
     )
 
 
-def _generic_list_cmd(cmd_args, select, opts, manager, print_fn):
+def print_diffs(
+    _manager,
+    selection=None,  # pylint: disable=unused-argument
```

Ruff (0.14.8)

Unused function argument: `selection`

(ARG001)